local.env

```diff
@@ -14,7 +14,6 @@ PRIVATE_KEYS=
 0x95eac372e0f0df3b43740fa780e62458b2d2cc32d6a440877f1cc2a9ad0c35cc,
 0x6c6e6727b40c8d4b616ab0d26af357af09337299f09c66704146e14236972106"
 
-RETRY_DELAY=3000 # ms
```

Comment:
SO do we still need this large polling logic or can this be reduced by default?

Reply:
We do, we just don't need it defined twice.